resources/constraints/constraints_py3.10.txt

```diff
@@ -1,5 +1,5 @@
 # This file was autogenerated by uv via the following command:
-#    uv pip compile --python-version 3.10 --output-file napari_repo/resources/constraints/constraints_py3.10.txt napari_repo/pyproject.toml napari_repo/resources/constraints/version_denylist.txt --extra pyqt5 --extra pyqt6_experimental --extra pyside2 --extra pyside6_experimental --extra testing --extra testing_extra --extra optional
+#    uv pip compile --python-platform aarch64-apple-darwin --python-version 3.10 --output-file napari_repo/resources/constraints/constraints_py3.10_macos_arm.txt napari_repo/pyproject.toml napari_repo/resources/constraints/version_denylist.txt --extra pyqt5 --extra pyqt6_experimental --extra pyside2 --extra pyside6_experimental --extra testing --extra testing_extra --extra optional
```

Comment:
I don't understand why the output file name does not match the name of this file...?

Reply:
(same comment for all the files.)

Reply:
Because this file only contains update of github actions and renaming:

1) constraints_py3.10.txt → constraints_py3.10_windows.txt
2) constraints_py3.10_macos_arm.txt → constraints_py3.10.txt

Without regenerating constraints. 

The comment will be fixed once `.github/workflows/upgrade_test_constraints.yml` will be executed after merging this PR. 

I want to keep PR size minimal (as #7045 shows that there will be a need to fix typing also). 

Should I fix this comments?

Reply:
ooooooh ok I get it! 😅 it's because git doesn't understand that the files have moved, it's just matching 3.10 with 3.10.

Um, it would be great if the comments could be updated but I am also ok with waiting for the next regeneration.

Another idea that would help keep things minimal: does uv pip compile also have the `--exclude-newer` option? If so, then these files could in fact be generated while keeping changes minimal?

Anyway, I'm ok with any choice — if you'd rather just merge that's fine. 🙏

Reply:
I prefer to just merge and regenerate